rulebook.md

```diff
@@ -367,9 +385,15 @@ Here are some examples scenarios and how they would be scored.
 ![flare](img/killswitch.jpg)
 <p class="image-caption"> Figure 16. Example a kill switch on an AUV from 2015 Competition.</p>
 
-
 ### Communications
-- No communication from/to the AUV of any sort is allowed during the game.
+
+- Untethered communications with the AUV **are allowed** in the main arena.
+- All commuication equipment on the AUV _MUST fit_ within the size limits for the AUV.
+- Any communication equipment (antenna, cable) on the AUV touching any part of the arena will be considered as the AUV touching that part of the arena, including water surface.
```

Comment:
We may need to differentiate between the communication equipment on the AUV and the communication equipment at the start arena. Otherwise, size limits, etc would be difficult to interpret.

Reply:
How about something like this? Is it more clear? Any better terms than "underwater" and "top-side" we can use here?

```
- Untethered communications with the AUV **are allowed** in the main arena.
- AUVs can carry their own "underwater" communication equipment.
- Teams are allowed to deploy "top-side" communication equipment in the starting zone during the game.
- All "underwater" communication equipment on the AUV _MUST fit_ within the size limits for the AUV.
- Any "underwater" communication equipment on the AUV is considered a part of the AUV.
- Any "top-side" communication equipment must stay within the starting zone. It may be submered in the water, and can have **a single tether** to the surface.
- Any "top-side" communication equipment submered in the water, must not leak and pollute the pool.
- Any "top-side" communication equipment must be removed at the end of the game.
```

Reply:
Add one more:
```
- The "top-side" communication equipment _MUST fit_ within the dimensions of the starting zone and weigh no more than XXX kg.
```
where `XXX` could be something similar to the weight of the AUV we allow?